src/main/resources/org/jenkinsci/plugins/codescene/CodeSceneBuilder/help-credentialsId.html

```diff
@@ -0,0 +1,2 @@
+Specify ID of Username/Password credentials defined in Jenkins configuration.
```

Comment:
I think it would be good with an example here on how such a configuration may look.